diktat-rules/src/test/resources/test/paragraph3/long_line/LongLineRValueTest.kt

```diff
@@ -0,0 +1,26 @@
+package test.paragraph3.long_line
+
+fun foo() {
+    val str = "This is very long string that should be split"
+
+    fun foo() {
+        val veryLoooooooooooooooooongNamesList = listOf<String>("Jack", "Nick")
+        veryLoooooooooooooooooongNamesList.forEach { name ->
+            if (name == "Nick") {
+                veryLoooooooooooooooooongNamesList.map { val str = "This string shouldn't be split"}
+                name.map { val str = "This string should be split" }
+            }
+        }
+    }
+
+    val longIntExpression = 12345 + 12345 + 12345 + 12345
+
+    val LongWithVar2 = "very long woooooordsdcsdcsdcsdc " +
+            "$variable"
```

Comment:
```suggestion
    val LongWithVar2 = "very long woooooordsdcsdcsdcsdc $variable"
```
Shouldn't it be on a single line for test?